pkgdb/src/buildenv/assets/mkContainer.nix

```diff
@@ -0,0 +1,26 @@
+# at this point just a thin wrapper around streamLayeredImage, but still worth
+# having to keep as much functionality as possible in lib rather than the module
+# system
+{
+  nixpkgsFlake, # the (bundled) nixpkgs flake,
+  environmentOutPath, # the path to the environment that was built previously
+  system, # the system to build for
+}: let
+  pkgs = nixpkgsFlake.legacyPackages.${system};
+  lowPriority = pkg: pkg.overrideAttrs (old: old // {meta = (old.meta or {}) // {priority = 10000;};});
+
+  buildLayeredImageArgs = {
+    name = "flox-env-container";
+    # symlinkJoin fails when drv contains a symlinked bin directory, so wrap in an additional buildEnv
+    contents = pkgs.buildEnv {
+      name = "contents";
+      paths = [
+        environmentOutPath
+        (lowPriority pkgs.bashInteractive) # for a usable shell
+        (lowPriority pkgs.coreutils) # for just the basic utils
+      ];
+    };
+    config = {};
```

Comment:
blocking: we should set entrypoint. Right now trying to run one of these containers gives:
```
docker run --rm -it 35440995c26a
docker: Error response from daemon: No command specified.
See 'docker run --help'.
```

See https://github.com/flox/floxpkgs/blob/master/lib/mkContainer.nix#L22 and https://github.com/flox/floxpkgs/blob/master/modules/container.nix#L115

Reply:
yeah i guess we'll want something like [flox/floxpkgs@master/modules/container.nix#L115](https://github.com/flox/floxpkgs/blob/master/modules/container.nix?rgh-link-date=2024-01-10T18%3A12%3A13Z#L115) for those